templates/cemanager.yaml

```diff
@@ -113,6 +113,10 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.namespace
+            - name: POD_IP
+              valueFrom:
+                fieldRef:
+                  fieldPath: status.podIP
```

Comment:
So this is so a replica behind an LB can connect to other replicas in the same cluster. What happens if this IP is a pod in a different cluster (like in a future where coderd is running multi-region)?

Reply:
Different clusters require a unique hostname setup for each. We can propagate this error really nicely to the user too!